problem1-1.py

```diff
@@ -0,0 +1,33 @@
+# Time Complexity : O(nlog(n))
```

Comment:
This can be done in O(logn) time complexity and O(1) space complexity using iterative binary search. Please try a bit more optimal solution. Also please do check the space and time complexities of this solution. You are performing binary search, and except recursive stack you did not use any extra auxiliary space